### PR TITLE
Ubuntu/bionic: Add Depends: python3-debconf

### PR DESCRIPTION
**Pull request by aciba90**
Depends: python3-debconf added in [#1506](https://github.com/canonical/cloud-init/pull/1506) and #1472

SC-1102

(Do not squash)

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,10 @@
+cloud-init (22.2-0ubuntu1~18.04.1) UNRELEASED; urgency=medium
```
Comment (blackboxsw) [inline debian/changelog:1]:
```suggestion
cloud-init (22.2-0ubuntu1~18.04.2) UNRELEASED; urgency=medium
```

File: debian/control [hunk 2]
```diff
@@ -36,6 +36,7 @@ Depends: cloud-guest-utils | cloud-utils,
          netplan.io | ifupdown,
          procps,
          python3,
+         python3-debconf,
```
Comment (blackboxsw) [inline debian/control:39]: We also need this in Build-Depends too. See https://github.com/canonical/cloud-init/pull/1511 for context.

File: debian/changelog [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+
+  * d/control:
+    - add Depends: python3-debconf
```
Comment (blackboxsw) [inline debian/changelog:4]:
```suggestion
  * d/control: add Depends: python3-debconf
```

**Review by blackboxsw — CHANGES_REQUESTED** [review]
ditto on the other packaging PRs. Thanks for working this @aciba90 we'll land tomorrow with the minor fixes.

**Comment by aciba90** [comment]
Thanks for the review @blackboxsw. Ready to be reviewed.

**Pull request merged**